### PR TITLE
bpo-45925: Update Windows installer to SQLite 3.37.2

### PR DESCRIPTION
**Pull request by kumaraditya303**
https://bugs.python.org/issue45925

### CODE REVIEW

**Comment by erlend-aasland** [comment]
Please split this up in one PR for the Windows installer and one for the macOS installer.

Also, I usually wait with these PRs until approx. a month has passed. For example with SQLite 3.35, there was a series of five (!) patch releases before that branch stabilised.

I suggest to wait with updating the installers until we've seen that the SQLite 3.37 branch has calmed down :) Keep an eye on the SQLite forum and on the SQLite SCM. If you see mentions of 3.37.3, you'll know there's an upcoming patch release.

**Comment by erlend-aasland** [comment]
The next 3.10 release is expected early February. I suggest to revisit updating the installers in approx. three weeks time.

**Comment by erlend-aasland** [comment]
Nit: its _macOS_, not _Mac_, and _SQLite_, not _sqlite_ 🤓

**Comment by erlend-aasland** [comment]
@kumaraditya303 Can you split up this in a Windows PR and a macOS PR?

**Comment by kumaraditya303** [comment]
@erlend-aasland Created https://github.com/python/cpython/pull/30921 Merge that first then I'll rebase.

**Comment by erlend-aasland** [comment]
> @erlend-aasland Created #30921 Merge that first then I'll rebase.

Please just remove the macOS part from this PR so the two teams can do their reviews independently; the Windows PR does not depend on the macOS PR, and the other way around. I'll remove the macOS team from the review list.

File: Mac/BuildScript/build-installer.py [hunk 1]
```diff
@@ -358,9 +358,9 @@ def library_recipes():
                   ),
           ),
           dict(
-              name="SQLite 3.36.0",
-              url="https://sqlite.org/2021/sqlite-autoconf-3360000.tar.gz",
-              checksum='f5752052fc5b8e1b539af86a3671eac7',
+              name="SQLite 3.37.2",
+              url="https://sqlite.org/2022/sqlite-autoconf-3370200.tar.gz",
+              checksum='683cc5312ee74e71079c14d24b7a6d27',
```
Comment (erlend-aasland) [inline Mac/BuildScript/build-installer.py]:
```suggestion
              name="SQLite 3.36.0",
              url="https://sqlite.org/2021/sqlite-autoconf-3360000.tar.gz",
              checksum='f5752052fc5b8e1b539af86a3671eac7',
```

File: Misc/NEWS.d/next/Build/2022-01-08-12-43-31.bpo-45925.38F3NO.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Updated Windows and Mac installers to sqlite 3.37.2.
```
Comment (erlend-aasland) [inline Misc/NEWS.d/next/Build/2022-01-08-12-43-31.bpo-45925.38F3NO.rst]:
```suggestion
Update Windows installer to use SQLite 3.37.2.
```

**Review by erlend-aasland — APPROVED** [review]
Thanks, LG

**Comment by zooba** [comment]
LGTM

**Comment by erlend-aasland** [comment]
Thanks for merging and for fixing the backports, Steve!

**Comment by erlend-aasland** [comment]
Thanks for the PR, Kumar!

**Pull request merged**